tutorials/evaluation.md

```diff
@@ -22,7 +22,6 @@ Use the following command to evaluate Lit-GPT models on all tasks in Eleuther AI
 python eval/lm_eval_harness.py \
     --checkpoint_dir "checkpoints/meta-llama/Llama-2-7b-hf" \
     --precision "bf16-true" \
-    --batch_size 4 \
```

Comment:
The generation does not support batching, so this was probably skipping 3 out of 4 samples